cdap-examples/Purchase/src/main/java/co/cask/cdap/examples/purchase/PurchaseApp.java

```diff
@@ -65,7 +65,16 @@ public void configure() {
     addService(new CatalogLookupService());
 
     // Schedule the workflow
-    scheduleWorkflow(Schedules.createTimeSchedule("DailySchedule", "", "0 4 * * *"), "PurchaseHistoryWorkflow");
+    scheduleWorkflow(Schedules.createTimeSchedule("DailySchedule", "Schedule execution every day", "0 4 * * *"),
```

Comment:
I think "daily schedule" already contain the information that it is a time schedule. And it says more: than it runs exactly once a day.